src/AWSS3.jl

```diff
@@ -32,6 +32,7 @@ using EzXML
 using Dates
 using Base64
 using UUIDs
+using Compat
```

Comment:
this is a dependency of AWS.jl anyway, so not a new dependency in the stack

Reply:
It's easy to lose track of why you needed to use Compat in the first place:
```suggestion
using Compat: @something
```